sympy/series/tests/test_formal.py

```diff
@@ -395,21 +393,20 @@ def test_fps__symbolic():
 
     f = x**(n - 2)*sin(x) + x**n*exp(x)
     assert fps(f, x).truncate() == \
-        (x**n*(1 + 1/x) + 5*x*x**n/6 + x**2*x**n/2 + 7*x**3*x**n/40 +
+        (x**n/x + x**n + 5*x*x**n/6 + x**2*x**n/2 + 7*x**3*x**n/40 +
          x**4*x**n/24 + 41*x**5*x**n/5040 + O(x**(n + 6), x))
 
     f = (x - 2)**n*log(1 + x)
     assert fps(f, x, 2).truncate() == \
-        ((x - 2)**n*log(3) - (x - 2)**2*(x - 2)**n/18 +
+        ((x - 2)**n*log(3) + (x - 2)*(x - 2)**n/3 - (x - 2)**2*(x - 2)**n/18 +
          (x - 2)**3*(x - 2)**n/81 - (x - 2)**4*(x - 2)**n/324 +
-         (x - 2)**5*(x - 2)**n/1215 + (x/3 - S(2)/3)*(x - 2)**n +
-         O((x - 2)**(n + 6), (x, 2)))
+         (x - 2)**5*(x - 2)**n/1215 + O((x - 2)**(n + 6), (x, 2)))
 
     f = x**n*atan(x)
     assert fps(f, x, oo).truncate() == \
-        (-x**n/(5*x**5) + x**n/(3*x**3) + x**n*(pi/2 - 1/x) +
-         O(x**(n - 6), (x, oo)))
-
+        (-(1/x)**n/(5*x**5) + (1/x)**n/(3*x**3) + (1/x)**n*(pi/2 - 1/x) +
+         O((1/x)**(n + 6), (x, oo)))
+        
```

Comment:
Here are the spaces causing the code quality test to fail..

Reply:
Done.